.gitignore

```diff
@@ -6,6 +6,8 @@ tmp
 .history
 dist
 /.idea
+.metals
+.bloop
```

Comment:
Please have these in your `~/.gitignore_global` instead.

Reply:
reverted